### PR TITLE
Update python versions (allow 3.7.5 and 3.8.0)

### PR DESCRIPTION
**Pull request by rodolfo3**

### CODE REVIEW

File: python/install [hunk 1]
```diff
@@ -45,9 +46,10 @@ apt-get remove --purge -y \
...
-apt-get install -y --no-install-recommends \
+apt-get install -y \
         libssl1.0.0 zlib1g libbz2-1.0 \
         libsqlite3-0 libncurses5 \
         libncursesw5 libreadline7
 
-apt-get autoremove -y
\ No newline at end of file
+apt-get autoremove -y
+apt-get clean
```
Comment (guilhermearaujo) [inline python/install]: If you clean, the image will be smaller, but it may take longer to install dependencies from the requirements.apt file, right?

**Comment by joaodaher** [comment]
Do we have `make` installed by default now? Can we remove from `requirements.apt`?

**Comment by guilhermearaujo** [comment]
I just rebased our master with the original source, now it supports python 3.7.4. This PR is no longer needed (even thought it does contain some additions)

**Comment by joaodaher** [comment]
https://github.com/edukorg/tsuru-platforms/pull/7 already solves this issue using the base repo code, not our own.

**Pull request closed without merging**